test/unit/cssnano-simple/cssnano-preset-simple/plugin.ts

```diff
@@ -12,7 +12,9 @@ const cssnanoPlugin = (options = {}) => {
   for (const nanoPlugin of nanoPlugins) {
     if (Array.isArray(nanoPlugin)) {
       let [processor, opts] = nanoPlugin
-      processor = processor.default || processor
+      processor =
+        // @ts-expect-error Property 'default' does not exist on type 'PluginCreator<any>'.
+        processor.default || processor
```

Comment:
Not changing runtime for safety. There's more runtime optimizations if we trust types so I'll do a follow-up so that we can easily revert if types are inaccurate.

Reply:
No longer needs changing but mostly because `cssnanoPresetSimple` doesn't have types when typechecked with the root tsconfig.